driver-adapters/neon-http-node-basic/index.js

```diff
@@ -2,7 +2,8 @@ const { Prisma, PrismaClient } = require('@prisma/client')
 const { neon } = require('@neondatabase/serverless')
 const { PrismaNeonHTTP } = require('@prisma/adapter-neon')
 
-const connectionString = process.env.DRIVER_ADAPTERS_NEON_HTTP_NODE_BASIC_DATABASE_URL
+const connectionString =
+  'postgres://prisma:pdYHi85xtOXa@ep-bold-meadow-77968096.us-east-1.aws.neon.tech/joel?connect_timeout=10'
```

Comment:
ooopsie

Reply:
Oopsie :D

Reply:
Maybe we need to revoke this secret now... 🙈